packages/formik/src/FieldArray.tsx

```diff
@@ -27,11 +27,11 @@ export type FieldArrayConfig = {
   /** Should field array validate the form AFTER array updates/changes? */
   validateOnChange?: boolean;
 } & SharedRenderProps<FieldArrayRenderProps>;
-export interface ArrayHelpers {
+export interface ArrayHelpers<T = void> {
```

Comment:
wouldn't it be more type-safe, if it had been `unknown[]`?

Reply:
```suggestion
export interface ArrayHelpers<T = unknown[]> {
```

Reply:
Yeah that's a great point, I'll update my PR. Thanks!